tests/vendor-show-test.js

```diff
@@ -29,11 +29,11 @@ describeApplication('VendorShow', function() {
     });
 
     it('displays the total number of packages', function() {
-      expect(VendorShowPage.numPackages).to.equal(`Total Packages${vendor.packagesTotal}`);
+      expect(VendorShowPage.numPackages).to.equal(`${vendor.packagesTotal}`);
```

Comment:
Well kinda... we either keep it in a string template or cast it with `toString()`.